modules/appeals_api/lib/appeals_api/higher_level_review_pdf_constructor.rb

```diff
@@ -21,7 +21,13 @@ def fill_pdf
     end
 
     def stamp_pdf(pdf_path, consumer_name)
-      bottom_stamped_path = CentralMail::DatestampPdf.new(pdf_path).run(text: 'API.VA.GOV', x: 5, y: 5)
+      stamper = CentralMail::DatestampPdf.new(pdf_path)
+      bottom_stamped_path = stamper.run(
+        text: "API.VA.GOV #{Time.zone.now.utc.strftime('%Y-%m-%d %H:%M%Z')}",
```

Comment:
this timestamp is being discussed
https://lighthouseva.slack.com/archives/CT0G8GMS7/p1588287510338500?thread_ts=1588271316.328000&cid=CT0G8GMS7
but I'm thinking my concerns relate more to the date that accompanies the signature